Xamarin.Forms.Core/Editor.cs

```diff
@@ -23,6 +23,8 @@ public class Editor : InputView, IEditorController, IFontElement, IPlaceholderEl
 
 		public static readonly BindableProperty PlaceholderColorProperty = PlaceholderElement.PlaceholderColorProperty;
 
+		public static readonly BindableProperty IsTextPredictionEnabledProperty = BindableProperty.Create(nameof(IsTextPredictionEnabled), typeof(bool), typeof(Editor), true, BindingMode.OneTime);
```

Comment:
Why set the default binding mode to `OneTime` rather than `Default`?

Reply:
According to documentation, the default will fallback to one way. So editor will be monitored for value changes. Is this behavior OK?

Reply:
Yes. I think the least surprising thing for users will be for it to behave like the other BindableProperties by default.

Reply:
Hi @hartez .
Would you take pick at my changes? Thanks.